frontend/packages/kubevirt-plugin/package.json

```diff
@@ -42,7 +42,7 @@
       ]
     },
     "exposedModules": {
-      "standaloneConsole": "./src/components/connected-vm-console/vm-console-page.tsx"
+      "standaloneConsole": "src/components/connected-vm-console/vm-console-page.tsx"
```

Comment:
This is just for consistency with existing relative paths in all static plugin `consolePlugin` meta objects.